config/initializers/good_job.rb

```diff
@@ -0,0 +1,23 @@
+# Prompt for a username if either the support user is defined, or we are in
+# production. In production, if no support user is defined then no access is
+# available.
+
+support_user_defined =
+  Settings.support_username.present? && Settings.support_password.present?
```

Comment:
Support user creds are currently set via `manifest.yml` and are hard-coded in git for prod. Can we move them to `SECRETS` instead? That should be done as a separate PR/deploy to first roll it out in every env, followed by this PR to add the `/good-job` dashboard.